doc/source/architecture.rst

```diff
@@ -45,9 +42,9 @@ or restore for that replica set.
 PBM Command Line Utility (|pbm.app|)
 ================================================================================
 
-|pbm.app| is the tool you will use hands on. It is a normal CLI for users and a
-command that can be executed periodically in scripts (for example, by
-``crond``). It manages your backups through a set of sub-commands:
+|pbm.app| is the command you will use manually in the shell, and it will also
```

Comment:
Higher up in this section you referred to |pbm.app| as a tool; in general, this paragraph provides the information that most CLI users already assume;